<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>folienizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="commentWindow">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="SearchForm" name="searchWindow" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File...</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom...</string>
    </property>
    <addaction name="actionFit_page_width"/>
    <addaction name="actionFit_page_height"/>
    <addaction name="action50"/>
    <addaction name="action100"/>
    <addaction name="action150"/>
    <addaction name="action200"/>
   </widget>
   <widget class="QMenu" name="menuSidebar">
    <property name="title">
     <string>Sidebar...</string>
    </property>
    <addaction name="actionTOC"/>
    <addaction name="actionBookmarks"/>
    <addaction name="actionLectures"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuZoom"/>
   <addaction name="menuSidebar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFit_page_width">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit page width</string>
   </property>
  </action>
  <action name="actionFit_page_height">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit page height</string>
   </property>
  </action>
  <action name="action50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50%</string>
   </property>
  </action>
  <action name="action100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="action150">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>200%</string>
   </property>
  </action>
  <action name="action200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>400%</string>
   </property>
  </action>
  <action name="actionTOC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>TOC</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionBookmarks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Bookmarks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionLectures">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lectures</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SearchForm</class>
   <extends>QWidget</extends>
   <header>searchform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
